nix/patches/prevent-install-and-copy.patch

```diff
@@ -1,88 +1,73 @@
 diff --git a/auto_cpufreq/core.py b/auto_cpufreq/core.py
-index 99397a9..16869ab 100755
```

Comment:
@shadeyg56 if you could also check Nix part since I don't use it that would be great, thanks!